app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,25 @@
+<section class="flex-center">
+
+<div class="form">
+
+<%= bootstrap_form_for(resource, as: resource_name, url: session_path(resource_name), layout: :horizontal, label_col: "col-sm-2", control_col: "col-sm-10") do |f| %>
+
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "bob@example.com" %></br>
+
+    <%= f.password_field :password, autocomplete: "current-password" %></br>
+    
+    <%= f.form_group do %>
+      <% if devise_mapping.rememberable? %>
+      <%= f.check_box :remember_me, switch: true %>
+      <% end %></br>
+    <% end %>
+
+    <%= f.form_group do %>
+      <%= f.button "Log in" %>
+    <% end %></br>
+<% end %>
+
+
+</div>
```

Comment:
- [ ] Just as the previous reviewer suggested, please let's make the login and register pages follow the wireframe guidelines:
1. Ensure that the login and register forms take up a considerable portion of the screen as shown in the wireframe.
2. Also ensure that the width of the  login and sign up buttons match that of the input fields.
### Wireframe Guidelines
![image](https://github.com/mudasiri/mybloggerapp/assets/46839548/7710162e-9f98-4f9a-b112-bf6049f7bd72)
### Your Design
![image](https://github.com/mudasiri/mybloggerapp/assets/46839548/1daf9291-680e-44ca-b7e1-1059bc0ca081)